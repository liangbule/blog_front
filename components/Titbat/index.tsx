import React from 'react';

const Index = (props: any) => (
    <div>我是but组件</div>
);

export default Index;