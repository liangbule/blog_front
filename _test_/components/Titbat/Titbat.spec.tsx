import renderer from 'react-test-renderer';
import Titbat from '../../../components/Titbat';

test('测试 Titbat 组件', () => {
    const tree = renderer.create(<Titbat/>).toJSON();
    expect(tree).toMatchSnapshot();
})
