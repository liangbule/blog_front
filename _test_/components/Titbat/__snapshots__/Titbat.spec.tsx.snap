// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 Titbat 组件 1`] = `
<div>
  我是but组件
</div>
`;
